index.html

```diff
@@ -0,0 +1,168 @@
+      stasis, Anadol faces all the new challenges that the gradual development
+      of an enriched immersive environment and ubiquitous computing impose on
+      architects, media artists and engineers.
+    </p>
+    <p>
+      How is our experience of space changing, now that digital objects ranging
+      from smart phones to urban screens have all but colonized our everyday
+      lives? How have media technologies changed our conceptualizations of
+      space, and how has architecture embraced these shifting
+      conceptualizations? These are the three main questions that Anadol tackles
+      by not simply integrating media into built forms, but by translating the
+      logic of a new media technology into spatial design
+    </p>
+
+    <p>Name: Carrie Roy</p>
+    <p>
+      Bio: Carrie grew up on a cattle ranch in southeast North Dakota and
+      received her BA from Harvard in Visual and Environmental Studies. Carrie
+      forges art that bridges the capabilities of computers and the creative
+      touch of humans. Her humanities research explores turning complex works of
+      human expression into numbers.
+    </p>
+    <p>Name: Skylar Tibbits</p>
+    <p>
+      Bio: Skylar Tibbits is a designer and computer scientist whose research
+      focuses on developing self-assembly and programmable materials within the
+      built environment.
+    </p>
+
+    <p>
+      Tibbits is the founder and co-director of the Self-Assembly Lab at MIT,
+      and Associate Professor of Design Research in the Department of
+      Architecture.
+    </p>
+
+    <p>
+      Tibbits has a professional degree in architecture and a minor in
+      experimental computation from Philadelphia University, and a masters in
+      design computation and masters computer science from MIT. He has worked at
+      a number of design offices including Zaha Hadid Architects, Asymptote
+      Architecture and Point b Design.
+    </p>
+
+    <p>
+      He has designed and built large-scale installations and exhibited in
+      galleries around the world, including the Centre Pompidou, Philadelphia
+      Museum of Art, Cooper Hewitt Smithsonian Design Museum, Victoria and
+      Albert Museum and various others. He is the author of the book
+      Self-Assembly Lab: Experiments in Programming Matter (Routledge, 2016),
+      Active Matter (MIT Press, 2017), co-editor of Being Material (MIT Press
+      2019) and the Editor-In-Chief of the journal 3D Printing and Additive
+      Manufacturing. Awards include LinkedIn's Next Wave Award for Top
+      Professionals under 35 (2016), R&D Innovator of the Year (2015), National
+      Geographic Emerging Explorer (2015), an Inaugural WIRED Fellowship (2014),
+      the Architectural League Prize (2013), Ars Electronica Next Idea Award
+      (2013), TED Senior Fellow (2012) and 2008 he was named a Revolutionary
+      Mind by SEED magazine.
+    </p>
+  </body>
+</html>
```

Comment:
Your current markup is looking good! See my additional comment so you know which elements are currently missing 👍🏻